libs/localization/src/lib/en.json

```diff
@@ -1841,18 +1843,24 @@
   "Forecasting": {
     "target": "Target",
     "whatIfHeader": "What-if analysis",
-    "whatIfDescription": "What-if allows you to perturb features for any input and observe how the model's prediction changes. You can perturb features manually or specify the desired prediction (e.g., class label for a classifier) to see a list of closest data points to the original input that would lead to the desired prediction. Also known as prediction counterfactuals, you can use them for exploring the relationships learnt by the model; understanding important, necessary features for the model's predictions; or debug edge-cases for the model. To start, choose input points from the data table or scatter plot.",
+    "forecastHeader": "Forecast analysis",
+    "whatIfDescription": "What-if allows you to perturb features for your entire time series and observe how the model's forecast changes.",
```

Comment:
should this be what-if analysis ? It seems a little weird to me just saying "what-if"

Reply:
or what-if forecasts

Reply:
We've been calling the overall feature "What If" for a long time, so I just went with that. That has no bearing on how they're rendered, of course. Following the convention here:
![image](https://user-images.githubusercontent.com/10245648/214614174-df005d2b-2337-4348-bf44-027275c8d8d4.png)

![image](https://user-images.githubusercontent.com/10245648/214614234-7b0dae9e-bacd-400c-813f-446d56d6b648.png)

![image](https://user-images.githubusercontent.com/10245648/214614335-9f56eb6b-d4c4-4282-b6be-4b7cd5046e95.png)

I do think What if forecasting may be useful to distinguish it from the "other" what if (the counterfactuals). Let me adjust that for the forecasting ones.